### ISSUE TITLE
next js app server error when emoji-mart init function

### ISSUE THREAD

**Issue by keven1024** [missive/emoji-mart#742]
when i use it
```
import {init} from "emoji-mart";
import data from '@emoji-mart/data'
init({data})
```
with next.js(ssr) , I get
![image](https://user-images.githubusercontent.com/99848979/203038335-9714bb07-1958-411d-8c5a-e9a7f0a5cbb9.png)

**Comment by phil-loops** [comment]
Potentially a duplicate of https://github.com/missive/emoji-mart/issues/735

**Comment by keven1024** [comment]
> Potentially a duplicate of #735

The solution to #735  does not apply to `init()`
I tried using `useEffect`, but I still get the error above.

**Comment by phil-loops** [comment]
The solution to 735 was lazily loading in the package before trying to run any of the functions. I'm noting that the ReferenceError in both cases is identical.

**Comment by EtienneLem** [comment]
Should be fixed in **v5.3.3**
